test/test_prototype_transforms.py

```diff
@@ -2043,3 +2045,102 @@ def test_classif_preset(image_type, label_type, dataset_return_type):
 
     assert out_image.shape[-2:] == (224, 224)
     assert out_label == label
+
+
+@pytest.mark.parametrize("image_type", (PIL.Image, torch.Tensor, datapoints.Image))
+@pytest.mark.parametrize("label_type", (torch.Tensor, list))
+@pytest.mark.parametrize("data_augmentation", ("hflip", "lsj", "multiscale", "ssd", "ssdlite"))
+@pytest.mark.parametrize("to_tensor", (transforms.ToTensor, transforms.ToImageTensor))
+def test_detection_preset(image_type, label_type, data_augmentation, to_tensor):
+    if data_augmentation == "hflip":
+        t = [
+            transforms.RandomHorizontalFlip(p=1),
+            to_tensor(),
+            transforms.ConvertImageDtype(torch.float),
+        ]
+    elif data_augmentation == "lsj":
+        t = [
+            transforms.ScaleJitter(target_size=(1024, 1024), antialias=True),
+            # Note: replaced FixedSizeCrop with RandomCrop, becuase we're
+            # leaving FixedSizeCrop in prototype for now, and it expects Label
+            # classes which we won't release yet.
+            # transforms.FixedSizeCrop(
+            #     size=(1024, 1024), fill=defaultdict(lambda: (123.0, 117.0, 104.0), {datapoints.Mask: 0})
+            # ),
+            # TODO: I have to set a very low size for the crop (30, 30),
+            # otherwise we'd get an error saying the crop is larger than the
+            # image. This means RandomCrop doesn't do the same thing as
+            # FixedSizeCrop and we need ot figure out the key differences
+            transforms.RandomCrop((30, 30)),
```

Comment:
`RandomCrop` doesn't do any padding by default, but `FixedSizedCrop` does. You probably only need to set `pad_if_needed=True`.